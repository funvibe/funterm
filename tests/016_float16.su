# Test Float 16 support
lua.float16_data = <<3.14:16/float, 2.5:16/float>>

match lua.float16_data {
    <<a:16/float, b:16/float>> -> {
        lua.print("Float 16-bit a:", a)
        lua.print("Float 16-bit b:", b)
    }
}