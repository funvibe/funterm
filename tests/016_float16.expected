Float 16-bit a: 3.140625
Float 16-bit b: 2.5